modules/core/src/lib/composite-layer.js

```diff
@@ -55,10 +55,11 @@ export default class CompositeLayer extends Layer {
   // override Layer.getPickingInfo() because decoding / setting uniform do
   // not apply to a composite layer.
   // @return null to cancel event
-  getPickingInfo({info}) {
+  getPickingInfo({info, sourceLayer}) {
```

Comment:
Nit: I assume we can't just add the sourceLayer to `info` from the caller?

This is a public `Layer` "lifecycle method"? Then we should probably document it.

Reply:
https://deck.gl/docs/api-reference/core/composite-layer#getpickinginfo